package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "productcat-widgets-javascript",
```

Comment:
pls change the name. This is not correct.

Reply:
changed